nixos/modules/programs/ssh.nix

```
@@ -27,6 +27,7 @@ let
     )) + "\n";
 
   knownHostsFiles = [ "/etc/ssh/ssh_known_hosts" ]
+    ++ map toString cfg.knownHostsPaths
```

```suggestion
    ++ map (p: "${p}") cfg.knownHostsPaths
```

This way it works correctly for path values, such as

```nix
knownHostsPaths = [
  ./known_hosts
];
```

Otherwise you might end up with paths like
```
GlobalKnownHostsFile /home/alice/src/config/foo/known_hosts
```
which won't be readable by other users, and won't be hermetic, impossible to deploy remotelyk, roll back, etc.